view/frontend/web/js/view/cart/minicart-updates.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Extend (c) 2023. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+define(['jquery', 'uiComponent', 'Magento_Customer/js/customer-data', 'extendSdk'], function (
+  $,
+  Component,
+  customerData,
+  Extend,
+) {
+  'use strict'
+  const minicartSelector = '[data-block="minicart"]'
+  const productItemSelector = '[data-role=product-item]'
+  const itemDetailsSelector = 'div.product-item-details'
+  const simpleOfferClass = 'extend-minicart-simple-offer'
+
+  const handleUpdate = function () {
+    const cartItems = customerData.get('cart')().items
+
+    cartItems.forEach(cartItem => {
+      const qtyElem = document.getElementById(`cart-item-${cartItem.item_id}-qty`)
+      if (qtyElem) {
+        const itemContainerElem = qtyElem.closest(productItemSelector)
```

Comment:
just curious, is this quantity value not on the cart item we get back from magento or do we do this in case they update it?

if it's the latter can we subscribe like we do elsewhere?
```
customerData.get('cart').subscribe(function (cart) {
```

Not an expert here, just curious 🙂

Reply:
ah looking further I see this happens on minicart update:
```
$(minicartSelector).on('contentUpdated', handleUpdate)
```
so we shouldn't need that subscription logic

Reply:
Yeah I was gonna throw a hissy fit if someone used meQuery #VanillaJSGang

Reply:
Removed some of the jQuery usage but was unable to get ride of the `on` call:
https://github.com/helloextend/magento-extend-integration/pull/60/files/fcb68e0b34a6880671267ef5ae6f530cef2d27f3#r1210892839